packages/react-sdk/README.md

```diff
@@ -691,7 +685,8 @@ Run `yarn dev` to build the SDK on changes and launch Storybook.
 
 - `yarn build`: Builds the SDK
 - `yarn clean`: Removes `node_modules`, `lib`, and `.turbo` folders
-- `yarn dev`: Launches Storybook for SDK components
+- `yarn dev`: Develop the `react-sdk` package
+- `yarn storybook`: Launches Storybook for SDK components at `http://localhost:6006`
```

Comment:
Thank you for this @rygine - ran it and it works for me. 

I wrote the description for `yarn dev` based on the `yarn dev` description for the `xmtp-web` level README. Is this correct? What does it mean?

Reply:
`yarn dev` will build the SDK and watch for code changes, which will trigger a re-build.